readme.md

```diff
@@ -173,6 +173,12 @@ const trustedDomains = {
 // scenarios which require it. For example, to send it via websocket connection.
 trustedDomains.config_show_access_token = { domains : ["https://demo.duendesoftware.com"], showAccessToken: true };
 
+// This example defines domains used by OIDC server separately from domains to which access tokens will be injected.
+trustedDomains.config_separate_oidc_access_token_domains = {
```

Comment:
hi @krzempekk , it woulob be nice to add this sample in multiAuthDemo page.

Reply:
Done